onnx/defs/math/defs.cc

```diff
@@ -1750,17 +1750,22 @@ void einsumRankInference(
 }
 
 static const char* Einsum_ver12_doc = R"DOC(
+An einsum of the form '''term1, term2, term3 -> output-term ''' produces an output tensor using the following equation
+
+'''output[i][j][k] = explicit-sum( input1[term1] * input2[term] * input3[term3] )'''
```

Comment:
Isn't the number of indices i, j, k and their order determined by output-term? So, it may be better to write "[output-term]" instead of "[i][j][k]", using the same notation as in "[term1]" etc. Alternatively, we should explain the relationship between "[i][j][k]" and output-term: this explanation is shared by "term1]" etc..

Reply:
It think now it looks fine. Except that the matrix multiplication in the equation is not very clear. In some cases, depending on the shape of the operands, this could be expand + multiply.